src/utils/axiosWithHeaders.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
```

Comment:
What I said before about this 👍 Or use this everywhere in all the handlers that we have right now or removing this and make a resetPasswordHandler that will use axios as we are using it in the other handlers.